joomla/joomla-cms
# [5.x] Corrected headings ordering in contact

Pull Request for Issue # .
If "Show Page Heading" in a single contact is set to "Hide", you will get errors with the accessibility checker, because the headings are skipping a level.

### Summary of Changes
Added a new `htag2 `variable to replace the hard-coded `<h3>` elements.


### Testing Instructions
Create a contact and a menu item for it.
Activate the Accessibility Checker Plugin in the frontend and check the output with and without Page Heading.


### Actual result BEFORE applying this Pull Request
Without Page Heading
![2023-07-10 15_58_10-Contact – Mozilla Firefox](https://github.com/joomla/joomla-cms/assets/9153168/a86f2ae6-3f02-467d-8cd7-95b3d80329d2)


### Expected result AFTER applying this Pull Request
With Page Heading
![2023-07-10 15_58_38-Contact – Mozilla Firefox](https://github.com/joomla/joomla-cms/assets/9153168/252fd6ea-8a58-40b6-97c6-bb028cb2be8c)

Without Page Heading
![2023-07-10 16_07_35-Contact – Mozilla Firefox](https://github.com/joomla/joomla-cms/assets/9153168/3b10c6c2-c00c-43e9-8ccb-046acd29e909)


### Link to documentations
Please select:
